rts/Lua/LuaSyncedRead.cpp

```diff
@@ -7165,6 +7168,44 @@ int LuaSyncedRead::GetFeaturePieceCollisionVolumeData(lua_State* L)
 }
 
 
+/*** Get the feature current fire timer in seconds.
+ *
+ * @function Spring.GetFeatureFireTime
+ *
+ * @param featureID integer
+ * @return number fireTime
```

Comment:
```suggestion
 * @return number fireTime in seconds
 ```
 
 Can omit "in seconds" from the header